assets/scss/modules/_datatables.scss

```diff
@@ -104,7 +104,11 @@
   .sorting_desc_disabled {
     background: url("../images/sort_desc_disabled.png") no-repeat scroll center right;
   }
-
+  
+  .sorting--hidden {
```

Comment:
Is this just hiding the sorting arrows? Would it be better to disable the functionality?